### ISSUE TITLE
Add search "nn"

### ISSUE THREAD

**Issue by torerobo** [sherlock-project/sherlock#479]
```
  "nnRU": {
    "errorType": "status_code",
    "rank": 12176,
    "url": "https://{}.www.nn.ru/",
    "urlMain": "https://https://www.nn.ru/",
    "username_claimed": "blue",
    "username_unclaimed": "noonewouldeverusethis7"
  },

```

**Comment by hoadlck** [comment]
The urlMain attribute does not work.

**Comment by torerobo** [comment]
> Атрибут urlMain не работает.
Sorry

ON
```
  "nnRU": {
    "errorType": "status_code",
    "rank": 12176,
    "url": "https://{}.www.nn.ru/",
    "urlMain": "https://https://www.nn.ru/",
    "username_claimed": "blue",
    "username_unclaimed": "noonewouldeverusethis7"
  },
```
Test
![1](https://user-images.githubusercontent.com/35532898/71625902-60db6c80-2bfb-11ea-9033-a115ca88241a.png)

**Comment by hoadlck** [comment]
Merged in #500
